### PR TITLE
getCitiesRouting new parameter street

### PR DESCRIPTION
**Pull request by jan-kratochvil**
New parameter street in getCitiesRouting API request

Hello, and thanks for contributing to PplMyApi

## tldr

There are two main goals in this document, depending on the nature of your PR:

- [description](#description): please tell us about your pr
- [checklist](#checklist): please review the checklist that is most closly related to your PR

The following sections provide more detail on each.

**Improve this document**

Please don't hesitate to [ask questions][issues] for clarification, or to [make suggestions][issues] (or a pull request) to improve this document.

## Description

To help the project's maintainers and community to quickly understand the nature of your pull requeset, please create a description that incorporates the following elements:

- [ ] what is accomplished by the PR
- [ ] if there is something potentially controversial in your PR, please take a moment to tell us about your choices

## Checklist

Please use the checklist that is most closely related to your PR _(you only need to use one checklist, and you can skip items that aren't applicable or don't make sense)_:

- [fixing typos]()
- [documentation]()
- [bug fix]()
- [new feature]()
- [other]()

### Fixing typos

- [ ] Please review the [readme advice]() section before submitting changes

### Documentation

- [ ] Please review the [readme advice](#readme-advice) section before submitting changes

### Bug Fix

- [ ] All existing unit tests are still passing (if applicable)
- [ ] Add new passing unit tests to cover the code introduced by your PR
- [ ] Update the readme (see [readme advice](#readme-advice))
- [ ] Update or add any necessary API documentation

### New Feature

- [ ] If this is a big feature with breaking changes, consider [opening an issue][issues] to discuss first. This is completely up to you, but please keep in mind that your pr might not be accepted.
- [ ] Run unit tests to ensure all existing tests are still passing
- [ ] Add new passing unit tests to cover the code introduced by your PR
- [ ] Update the readme (see [readme advice](#readme-advice))

Thanks for contributing!

## Readme advice

Please review this section if you are updating readme documentation.

**Code comments**

Please add code comments (following the same style as existing comments) to describe any code changes or new code introduced by your pull request.

[issues]: ../../issues

### CODE REVIEW

**Comment by Salamek** [comment]
@jan-kratochvil when i'm looking at `https://myapi.ppl.cz/MyAPI.svc?singleWsdl`, i can see much more parameters for `GetCitiesRouting`:

```xml
<xs:complexType name="CitiesRoutingFilter">
<xs:sequence>
<xs:element minOccurs="0" name="CountryCode" nillable="true" type="xs:string"/>
<xs:element minOccurs="0" name="DateFrom" nillable="true" type="xs:dateTime"/>
<xs:element minOccurs="0" name="EveningDelivery" nillable="true" type="xs:boolean"/>
<xs:element minOccurs="0" name="MorningDelivery" nillable="true" type="xs:boolean"/>
<xs:element minOccurs="0" name="PackProductType" nillable="true" type="xs:string"/>
<xs:element minOccurs="0" name="Post" nillable="true" type="xs:string"/>
<xs:element minOccurs="0" name="Rejected" nillable="true" type="xs:boolean"/>
<xs:element minOccurs="0" name="SaturdayDelivery" nillable="true" type="xs:boolean"/>
<xs:element minOccurs="0" name="Street" nillable="true" type="xs:string"/>
<xs:element minOccurs="0" name="ZipCode" nillable="true" type="xs:string"/>
</xs:sequence>
</xs:complexType>
```

Those are not yet used/documented/needed?

**Comment by jan-kratochvil** [comment]
Hi, I think they are not needed for label cretion or printing. The zip and street parameter are needed for routing label.
